public/js/eventList.js

```diff
@@ -0,0 +1,36 @@
+
+document.querySelectorAll(".event .lower").forEach(function(event) {
+    event.dataset.height = event.clientHeight
+    event.style.height = 0
+})
+
+document.querySelectorAll(".event").forEach(function(event) {
```

Comment:
Inconsistent use of ' & " throughout file.

Reply:
made all the quotes ''

Reply:
cb6f51e